content/user-guides/connector-guides/sink/http-setup.md

```diff
@@ -1 +1,49 @@
 # HTTP
+
+This guide contains information to set up a HTTP Sink in Vanus Cloud.
+
+## Introduction
+
+The HTTP Sink can send real-time Cloud Events to the expected targets using Webhook. It extracts the data part of the original event and deliver to the target URL.
+
+## Prerequisites
+
+Before sending HTTP events to its target URL, you must have:
+
+- A [Vanus Cloud account](https://cloud.vanus.ai)
+- A target URL
+
+## Getting Started
+
+Cloud Events in json format are sent to an HTTP server which extracts the `data` part of the event and send the `data` to it's target URL in Vanus Cloud. An example of an incoming Cloud Event looks like:
```

Comment:
That incorrect it takes the whole cloud event and put it in a HTTP request check the docs of the HTTP sink

Reply:
You shouldn't event mention that actually because it will only send what you put in the template

Reply:
There no how to use it